__tests__/app.test.js

```diff
@@ -64,4 +64,48 @@ describe('/api/articles/', () => {
             })
         })
     })
+    describe('PATCH', () => {
+        test('Status: 200 - accept an object of inc_votes which increments vote by a positive number', () => {
+            const positiveVote = {inc_votes: 1};
+            return request(app).patch(`/api/articles/2`).send(positiveVote).expect(200)
+            .then(({body}) => {
+                expect(body.article).toEqual(
+                    expect.objectContaining({
+                        article_id: 2,
+                        title: expect.any(String),
+                        topic: "mitch",
+                        author: "icellusedkars",
+                        body: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: 1
+                    })
+              )
+            })
+        })
+    
+    test('Status: 200 - accept an object of inc_votes which decrements vote by a negative number', () => {
+        const negativeVote = {inc_votes: -100};
+        return request(app).patch(`/api/articles/4`).send(negativeVote).expect(200)
+        .then(({body}) => {
+            expect(body.article).toEqual(
+                expect.objectContaining({
+                    article_id: 4,
+                    title: expect.any(String),
+                    topic: "mitch",
+                    author: "rogersop",
+                    body: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: -100
+                }))
+        })
+        
+    })
+
+    test('status: 400 - reponse with err msg for invalid inc_vote request', () => {
+        const invalidVote = {inc_votes: NaN}
+        return request(app).patch("/api/articles/3").send(invalidVote).expect(400).then(({ body: { msg } }) => {
+            expect(msg).toBe("Bad Request");
+        })
```

Comment:
tests - patch to an article that doesnt exist?

Reply:
added this test